openmdao/api.py

```diff
@@ -40,10 +39,8 @@
 from openmdao.solvers.run_once import RunOnce
 from openmdao.solvers.scipy_gmres import ScipyGMRES
 from openmdao.solvers.solver_base import LinearSolver, NonLinearSolver
-try:
+if under_mpirun():
     from openmdao.solvers.petsc_ksp import PetscKSP
```

Comment:
Could we just get away with generalizing the except clause, i.e., use 'except Exception' instead of 'except ImportError'.?